### PR TITLE
Change main view to haml

### PR DESCRIPTION
**Pull request by Angel-cuba**

### CODE REVIEW

File: app/views/layouts/application.html.haml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    = stylesheet_link_tag "application", "data-turbo-track": "reload" 
+    = javascript_importmap_tags 
+
+  %body
+    %p
+      .notice
+      = notice 
+    %p 
+      .alert
+      = alert
```
Comment (valscion) [inline app/views/layouts/application.html.haml:18]: Can you check that these notices and alerts are showing up in the correct place? For example, a notice should show when you log in and an alert should show when you try to do something you're not authorized to do.

I think one thing that you are unauthorized to do is trying to visit the "add new game" page when you are logged out.

File: app/views/layouts/application.html.haml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+!!!
+%html
+  %head
+    %title = VideoGameDatabase
```
Comment (valscion) [inline app/views/layouts/application.html.haml:4]: Can you check the browser tab title if this is as expected?

File: app/views/layouts/application.html.haml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    %meta{:charset => "utf-8"}
```
Comment (valscion) [inline app/views/layouts/application.html.haml:5]: We didn't have this `charset` tag in the ERB side so let's not add it to the HAML side, either.

File: app/views/layouts/application.html.haml (hunk 1 above)
Comment (Angel-cuba) [reply app/views/layouts/application.html.haml:18]: To show `notice`, we were using the default position, in the left-top corner. Here we can see how the `notice` looks like
**User has logged in**
signed_in

**When has signed out**
sign_out

For `alert`, i changed the value of `info` function to `false` and this how looks like if we are not allowed to see what is inside the info view
alert_message

`Note:` We decided to delete `add new game` from the **navbar** if the user wasn't logged in

File: app/views/layouts/application.html.haml (hunk 3 above)
Comment (Angel-cuba) [reply app/views/layouts/application.html.haml:5]: I think, i added this line following some example from `Stack Overflow` 🤔.

File: app/views/layouts/application.html.haml (hunk 1 above)
Comment (valscion) [reply app/views/layouts/application.html.haml:17]: Cool! Can you check how the same UI looks like if you run them in `main` branch to see how the same steps used to look like over there?

**Comment by valscion** [comment]
I think there's a behavior change in this PR compared to how it used to be. Please check my comment here: https://github.com/venuu/video-game-database-ruby/pull/36#discussion_r1053369827

File: app/views/layouts/application.html.haml (hunk 1 above)
Comment (Angel-cuba) [reply app/views/layouts/application.html.haml:17]: If i run them in `main`, we'll have same result regarding to the position, but with some styles. Same views look like this
**Notice**
`User signed in`
Screenshot 2022-12-20 at 16 40 26

`User signed out`
Screenshot 2022-12-20 at 16 40 43

**Alert**
Screenshot 2022-12-20 at 16 40 57

File: app/views/layouts/application.html.haml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    %meta{:name=> "viewport", :content => "width=device-width, initial-scale=1"}
+    = csrf_meta_tags
+    = csp_meta_tag 
+    = stylesheet_link_tag "application", "data-turbo-track": "reload" 
+    = javascript_importmap_tags 
+
+  %body
+    .notice
+      %p
+      = notice
```
Comment (valscion) [inline app/views/layouts/application.html.haml]:
```
<div class="notice">
<p></p>
You are now logged in
</div>
```

File: app/views/layouts/application.html.erb [hunk 5]
```diff
@@ -1,19 +0,0 @@
...
-    <meta name="viewport" content="width=device-width,initial-scale=1">
-    <%= csrf_meta_tags %>
-    <%= csp_meta_tag %>
-
-    <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
-    <%= javascript_importmap_tags %>
-  </head>
-
-  <body>
-    <p class="notice"><%= notice %></p>
```
Comment (valscion) [inline app/views/layouts/application.html.erb:14]:
```
<p class="notice">You are now logged in</p>
```

**Review by valscion — APPROVED** [review]
:+1:

**Pull request merged**